operator/roles/forkliftcontroller/tasks/main.yml

```diff
@@ -80,6 +80,16 @@
       definition: "{{ lookup('template', 'controller/provider-host.yml.j2') }}"
     when: "'kubevirt.io' in api_groups"
 
+  - name: "Setup volume-data-source-validator deployment"
+    k8s:
+      state: present
+      definition: "{{ lookup('template', 'ovirt-populator/deployment-data-source-validator.yml.j2') }}"
+
+  - name: "Setup ovirt-populator deployment"
+    k8s:
+      state: present
+      definition: "{{ lookup('template', 'ovirt-populator/deployment-ovirt-populator.yml.j2') }}"
```

Comment:
shouldn't the populator be like the conversion pods we have with virt-v2v, why do we need a deployment?

Reply:
so we should create a pod each time we wish to transfer disk and remove it afterwards? there is already a pod from populator that doing the actual "job" and removed once the transfer finishes..

Reply:
> there is already a pod from populator that doing the actual "job" and removed once the transfer finishes..

ok, I thought you referred to this one - so you mean the controller?

Reply:
we have a pod that listen and once triggered it calls the volume populator - which creates another pod (as we had importer pod per disk).

i'm not sure what you exactly mean

Reply:
ok, what you described above is the controller (so we can call it populator-controller to distinguish it from 'populators' which corresponds to the "other pod" you mentioned above) - that needs a deployment, ok

Reply:
@bennyz this does act as controller. do you plan to use the same pod for OSP? or it should be distinguished to ovirt?

Reply:
openstack will have a separate pod